src/components/Tasks/TaskStatus.js

```diff
@@ -16,6 +16,22 @@ function getDueDayText(dueDate) {
 }
 
 const TaskStatus = React.memo(function TaskStatus({ dueDate }) {
+  const [refresh, setRefresh] = useState(false)
+
+  const now = new Date()
+  const refreshTime = dueDate - now
```

Comment:
Since the refreshing is relative to the remaining days we should calculate this as 

`const refreshTime = (dueDate - now) % DAY_IN_SECONDS` where `DAY_IN_SECONDS = 86400`

Reply:
I think that the timeout should be until local `00:00 hs` and then that number is updated

Reply:
Yes totally true 👍